lib/payloads.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const defaultPayload = {
+  timestamp: undefined
+, line: undefined
+, level: undefined
+, app: undefined
+, env: undefined
+, meta: undefined
+}
+
+const agentPayload = {
+  t: undefined
```

Comment:
`t` is short version for `timestamp` here - so, you can use `timestamp` to look more descriptive

Reply:
Same.  Would love to change it, but I think it's dangerous until we're sure the agent works with these changes.